waiter/src/waiter/token.clj

```diff
@@ -719,3 +760,128 @@
                                              :status http-405-method-not-allowed})))
     (catch Exception ex
       (utils/exception->response ex req))))
+
+; TODO:
+; aggregate events,
+; possibly aggregate from tokens-event-chan,
+; finish migrating token-watch-tests
+(defn start-token-watch-maintainer
```

Comment:
I'm thinking whether we should create a new `token_watch.clj` file for the watch maintainer.